utils/zerovec/Cargo.toml

```diff
@@ -28,7 +28,7 @@ zerofrom = { workspace = true }
 zerovec-derive = { workspace = true, optional = true}
 
 databake = { workspace = true, features = ["derive"], optional = true }
-serde = { workspace = true, features = ["alloc"], optional = true }
+serde = { workspace = true, features = ["alloc", "derive"], optional = true }
```

Comment:
Issue: I'm not very excited about `zerovec` gaining a serde_derive dependency... but I guess since it already has a databake/derive dependency, the harm is limited